.gitlab-ci.yml

```diff
@@ -88,12 +88,6 @@ cargo-test-af:
   script:
   - cargo test --verbose --all --all-features
 
-cargo-build-nf:
-  stage:                           check-workspace
-  <<:                              *docker-env
-  script:
-  - cargo build --verbose --all --no-default-features --release --target=wasm32-unknown-unknown
-
```

Comment:
So you fix it by removing it?

Reply:
`net2` will build error on target `wasm32-unknown-unknwon`